lib/substitutions.json

```diff
@@ -15,6 +15,11 @@
       "before": "each others houses",
       "after": "each others’ houses"
     },
+    {
+      "before:" "My house is old,",
+      "after": "My house was old,",
+      "_comment:" "The tense doesn't match the rest of the text."
```

Comment:
I'll revert this. It's fine English to say that something is X, and something else was Y. Tenses don't have to match for everything in a sentence.

Other nearby sentences talking about the house are also in present tense, e.g. "The walls and floor **are** concrete"

Reply:
I wasn't very sure about this one, thanks for the clarification.